Rise/UI/Views/TabBarController.swift

```diff
@@ -35,6 +35,8 @@ final class TabBarController: UITabBarController, UITabBarControllerDelegate {
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        
+        registerLocal()
```

Comment:
Думаю нам лучше завязаться на онбординг, показывать запрос на permissions после прохождения онбординга, это достаточно часто используется в других приложениях
Это можно сделать в `RootCoordinator.onboarding.finish`, строка 73

Reply:
<img width="943" alt="Снимок экрана 2023-06-03 в 21 52 07" src="https://github.com/VladimirBrejcha/Rise/assets/117162015/39d7eb4e-3663-46cd-bd75-7fa6bb95bdbf">
Переместил

Reply:
Согласно рекомендациям apple (статья, которую ты скинул), лучше вызывать запрос (на уведомления) после действия юзера, например - поставил себе задачу для сна ( Sending the request in context provides a better user experience than automatically requesting authorization on first launch, because the user can more easily see what purpose the notifications serve.). Что думаешь?

Reply:
Звучит здраво, можно попробовать в таком случае завязаться на два места:
1. Пользователь создал режим сна (ввел все данные и нажал создать)
2. Пользователь начал сон через кнопку sleep

Давай сделаем в рамках этой задачи
Если нужен созвон чтобы разобраться где это лучше вызывать — дай знать